packages/kubernetes-1.20/kubelet-sysctl.conf

```diff
@@ -0,0 +1,4 @@
+# Ports below 32768 are generally outside of the ephemeral port range
+# Aligning with the port range documented for ECS
+# https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_PortMapping.html
```

Comment:
```suggestion
# This is generally considered a safe ephemeral port range
```
I'm thinking it may be confusing to point to ECS documentation in a k8s variant, and it seems fine to just have a comment about choosing a safe range?  (Also, based on the linked doc, it seems the range has changed, and it may be confusing what's intended.)